src/chain/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Validation } from 'src/validation/index'
+import { ArenaObject } from 'src/common'
+import { ProcessingStep } from './processingStep'
+import { Labels } from 'src/labels/index'
+
+export enum StatusExec {
+  error = 'error',
+  success = 'success',
+  running = 'running',
+}
+
+export interface ChainProps {
+  cycles: Array<string>
+  descriptions: { [key: string]: string }
```

Comment:
don't we have a common Labels object?

Reply:
Should `descriptions` be `Labels` too ?
or Descriptions?